filebeat/module/elasticsearch/slowlog/manifest.yml

```diff
@@ -0,0 +1,13 @@
+module_version: 1.0
+
+var:
+  - name: paths
+    default:
+      - /var/log/elasticsearch/*_index_search_slowlog.log
+    os.darwin:
+      - /usr/local/elasticsearch/*_index_search_slowlog.log
```

Comment:
What is the value of the `*` here? Does it contain info like cluster id or node id?